src/layouts/MainLayoutHeader.vue

```diff
@@ -355,6 +358,16 @@ export default {
         </QInput>
       </div>
 
+      <div class="q-mx-md col-md-9">
+        <div
+          v-show="isRecruiters"
+          :class="{ 'recruiters-title': isRecruiters }"
+          class="text-warning text-weight-medium text-center text-uppercase text-subtitle1"
+        >
```

Comment:
Broken header is probably due to your `v-show` being on the child of the div you added, rather than on parent:

```js
<div class="q-mx-md col-md-9">
        <div
          v-show="isRecruiters"
          :class="{ 'recruiters-title': isRecruiters }"
          class="text-warning text-weight-medium text-center text-uppercase text-subtitle1"
        >
        {{ $t({ id: 'pages.recruiters.tagline' }) }}
       </div>
</div>
```

Reply:
Okay, I changed it. I'm waiting for a feedback before commit it